### DISCUSSION TITLE
✅ 🔥 Elementor v3.17 Beta Release 🔥

### DISCUSSION THREAD

**Discussion by avivu** [elementor/elementor#23925] [Announcement]
===========
# [Beta 3/4 UPDATED - PLEASE READ BELOW!](https://github.com/orgs/elementor/discussions/23925#discussioncomment-7316281)
===========

<img width="1200" alt="cover-3 17" src="https://github.com/elementor/elementor/assets/76733085/5a7c0be3-c060-4fb5-9b92-bd1ebf638d59">

&nbsp;

**Hey Beta testers,** 👋

The beta version of Elementor 3.17 is now available, and we're eager to hear your feedback on the new features. The anticipated AJAX Loading feature is now available for Loop Grids, which enables visitors to load loop content quickly and efficiently without the need to reload the entire page.

Additionally, Elementor AI now has a history panel, so you can access prompts and content used in the past. This version also includes performance enhancement experiments, a new Rating Widget and improvements to the Accordion Widget.

We encourage you to try out this beta version and provide us with your thoughts on our latest additions, which include:

- **[PRO] AJAX Loading for Loop Grids (4th most requested)**
- **[Elementor AI] AI History**
- **Performance Enhancements**
- **New Rating Widget and More!**

&nbsp;

# [PRO] Introducing AJAX for Loop Grids - For Seamless Content Loading

<img width="1200" alt="Ajax Pagination" src="https://github.com/elementor/elementor/assets/76733085/4d93f96c-cbb5-4a4f-9464-53d04adcc7f4">

&nbsp;

We are excited to introduce the fourth most requested feature on GitHub: AJAX loading for Loop Grids. AJAX, short for Asynchronous JavaScript and XML, allows visitors to load new content into a page without having to reload the entire page.

## AJAX Loop Grid Pagination

Visitors no longer need to reload the entire page when switching between the pages of your listing. Instead, clicking on the Loop Grid’s pagination element will only reload the Grid's content. This creates a smoother experience for visitors. AJAX Pagination also gives each page of your listing its own URL parameter, so visitors can easily share links to listing pages.
 
As part of this feature, you can also enable the "Autoscroll" control. That way, when visitors change pages of your listing, the browser will automatically scroll to the top of the grid. If you are using a taxonomy filter and want to make sure it remains in view, you can input an offset value.
 

#### How to Test It:

1. Ensure your website has at least 12 posts or products.
2. Create a new page and edit it using Elementor.
3. Drag the Loop Grid widget into the editor.
4. Select or create a new Loop Item template.
5. In the Pagination section, choose the "Numbers" option.
6. From the "Load Type" dropdown, select the "AJAX" option.
7. Publish the page and preview it on the front end.
8. Try navigating between pages and confirm that the pagination operates without reloading the entire page, using AJAX functionality.
9. Verify that everything functions as expected in the Editor and the front end.

## Individual Pagination for Loop Grid and Post Widgets

In previous versions, changing the pagination of a listing would affect the content displayed on all other Loop Grid or Post Widgets on the same page. For example, selecting the third page of a listing would also load the third page of other listings if both were placed on the same page. However, with the Individual Pagination feature, visitors can now change the pagination of a listing without affecting other Loop Grids and Posts Widgets on the page.

To assign individual loading settings, ensure that you have enabled the "Individual Pagination" control under Content Tab > Pagination of each Loop Grid and Posts Widget on the page.

#### How to Test It:

1. Ensure your website has at least six posts or products.
2. Create a new page and edit it with Elementor.
3. Add either the Loop Grid or Posts widget to the editor.
4. Choose or create a new Loop Item template.
5. Under the Pagination section, select the "Numbers" option.
6. In the "Load Type" dropdown, use the "Page Reload" option.
7. Enable the "Individual Pagination" option.
8. Duplicate the Loop Grid widget and ensure the "Individual Pagination" option is enabled for the duplicate.
9. Publish the page and preview it on the front end.
10. Test navigating between pages and confirm that navigating within one Loop Grid does not affect the other, and vice versa.
11. Verify that everything functions as expected in the Editor and front end.

&nbsp;

# [Elementor AI] AI History - Restore AI Generated Content

<img width="1200" alt="AI history" src="https://github.com/elementor/elementor/assets/76733085/c0fe80e0-6b45-4865-b865-15d03907ffe5">

&nbsp;

Elementor 3.17 enhances your workflow on Elementor AI with the AI History. This feature displays previously used prompts and the images or texts they generated. Restoring prompts and their generated content allows you to pick up where you left off. Continue editing the content with AI, use it on your site, or permanently delete them from AI History. While you will not be able to get previously generated code from the AI History, since the code is generated for specific widgets, you can access prompts used to generate them to regenerate new code from your historical prompts. With the AI Starter plan, you can view and restore content generated up to 30 days ago, and with the AI Power plan, you’ll be able to access content generated up to 90 days ago.

&nbsp;

# Performance Improvements for Faster LCP

<img width="1200" alt="Performance" src="https://github.com/elementor/elementor/assets/76733085/e0d378e0-8c6d-426b-9996-cd27ebd20aff">

&nbsp;

Elementor 3.17 makes your site even faster with two features that will turbocharge your website’s loading time.

## Optimize Image Loading

Elementor has partnered with Chrome to identify opportunities for improving the performance of websites built with Elementor. This collaboration involves utilizing Google tools such as the Chrome User Experience Report and Core Web Vitals Technology Report. One of the results of this collaboration is Google’s decision to contribute the “Optimize image loading” features to Elementor's core plugin as part of their ongoing commitment of making the web better performing.

This experiment improves frontend speed by prioritizing image loading based on the placement of the images on the page. It detects which image is most likely to be the "LCP image", which refers to the largest contentful paint element in the viewport, and prioritizes its load by using priority hints.

Additionally, the experiment applies lazy-loading to images that are not initially displayed in the viewport and loads them when they come into view. These changes improve LCP by 5-10% depending on the complexity of the page. Optimize Image Loading also works with Elementor websites running on older versions of WordPress that don’t include LCP detection updates.
 
Activate the “Optimize Image Loading” features by going to the Elementor menu in WordPress → Elementor → Settings → Features.

## Gutenberg Asset Loading

Optimize your site’s frontend speed by excluding Guttenberg styles and scripts when not in use. This experiment will improve your site’s LCP (Largest Contentful Paint) Core Web Vital metric by detecting when Guttenberg wasn’t used to create a page or generate its content. This allows Elementor to avoid loading unnecessary code. 

Activate the “Gutenberg Asset Loading” experiment by going to the Elementor menu in WordPress → Elementor → Settings → Features.

&nbsp;

# New Rating Widget For Design Flexibility

<img width="1200" alt="rating-widget" src="https://github.com/elementor/elementor/assets/76733085/83fcd882-d927-4cc9-9d77-61f3f4e555e3">

&nbsp;

This version includes the new Rating Widget, which gives you more flexibility to customize your visual rating scales for products, content, and more. To visually represent your rating, you can choose between Font Awesome icons or upload your own SVG.

The Rating Widget expands the options of rating scales that were previously available in the Star Rating Widget. Instead of choosing between a rating scale of 0-5 or 0-10, you can set a rating scale from zero to any number that fits your needs. Additionally, the Rating Scale dropdown has been replaced with a range.

The Rating Widget also enhances your site's SEO ranking with an improved schema attribute. The Rating Widget replaces the Star Rating Widget in the Widget Panel. However, any Star Ratings already placed on your site will not be affected.

#### How to Test It:

1. Create a new page and edit it using Elementor.
2. Ensure that any existing Star Rating widgets within the page are editable and that you can customize them.
3. Drag the Rating widget into the editor.
4. Adjust the Rating Scale and Rating controls, ensuring the rating works as expected.
5. Experiment with the Icon control, where you can choose from the Font Awesome library or SVG icons.
6. Head over to the Style tab and adjust the icon's Size, Spacing, and Colors.
7. Publish the page and preview it on the front end.
8. Verify that everything functions as expected in the Editor and front end.

&nbsp;

# Accordion Improvement - Make Your FAQ Frequently Found

<img width="1200" alt="Accordion" src="https://github.com/elementor/elementor/assets/76733085/1a3af99e-9f1d-496c-882e-86ba74e1bfab">

&nbsp;

The Accordion Widget, with its nested capabilities, allows you to place Widgets inside Accordion items, giving you unlimited layout options. In version 3.17, the Accordion Widget also improves the SEO compatibility of your content by adding a FAQ Schema control.

The FAQ Schema adds a specialized markup to your page’s code that contains a list of questions and answers. Search engines can read this markup and use it to generate rich code.

&nbsp;

# Accessibility Enhancements

<img width="1199" alt="a11y" src="https://github.com/elementor/elementor/assets/76733085/04b5cc44-7df1-4800-a4b3-37560003042c">

&nbsp;

- **Popups:** Improve focus management when enabling the Accessible Navigation option, by focusing initially on the popup close button instead of other focusable elements inside popups.
- **Table of Contents**: Indicate that the minimize/maximize button in the Table of Contents widget is an interactive element for visitors using assistive technologies.
- **Nested Accordion:** Accordion layouts are now accessible to visitors using keyboard navigation.

&nbsp;

# Additional Updates

<img width="1200" alt="features" src="https://github.com/elementor/elementor/assets/76733085/f067ce0e-5ef2-4f3e-bf64-8e1a9f7ab3ad">

&nbsp;

1. **Menu widget Enhancements:** Prevent menus from transforming into a dropdown layout, select the "None" option from the list of breakpoints. If the menu is wider than the viewport, you can enable horizontal scrolling to maintain its original inline design. This will allow visitors to scroll horizontally and prevent the menu from breaking into two lines.
2. **VideoPress Integration:** Seamlessly integrate VideoPress into your Elementor site. Enjoy all the features VideoPress offers, such as self hosted videos, transcoded video file for playback on many device types, adaptive streaming based on device screen size and available bandwidth, the ability to authenticate private videos, and more.

&nbsp;

# Responsive Template Designs

This version has improved your ability to correctly display changes made to responsive designs of templates with additional breakpoints. You will no longer need to regenerate CSS to see changes you made to templates when editing in the theme builder. Changes will be reflected in your website's editor and frontend.

#### How to Test It:

**Important: This test is focused on responsive breakpoints and templates. Please refrain from using the "Regenerate CSS" option during testing, and ensure that everything works without it.**

1. Activate the "Additional Custom Breakpoints" feature, enabling all seven breakpoints.
2. Create a new page and insert a Loop Grid widget into it.
3. In the Theme Builder, create a Loop Item Template. **Important:** Instead of using in-place editing, open the template in a new tab.
4. Insert a Template widget onto the page and select a container template created previously.
5. Make sure you have both a Header and Footer on this page.
6. Open all templates in new tabs: Header, Footer, Loop Item, and Container Template.
Edit these templates, adjusting settings and values for Desktop and Mobile devices separately.
7. Save all the templates after making your adjustments.
8. After saving, check the Page, refresh it, and verify that all the settings you applied to the templates for different devices are correctly applied to the templates within the page.
9. Check these templates on the front end to ensure all values are displayed as expected.
10. For an additional scenario, edit your Single Post template and, after saving, check the templates within the initial page to ensure all values are presented as intended.
11. Verify that everything functions as expected in the Editor and front end.

&nbsp;

----
# More Improvements

#### Free version

* New: AI History - Quickly restore AI-generated content, enhancing your workflow by improving content recovery and acceleration
* New: Revamped Rating widget - enhanced flexibility for customizable visual rating scales, elevating your designs to be more engaging and interactive (#18793, #10529, #18169, #20651, #10127)
* New: Introducing Gutenberg Asset Loading as a Beta feature - enhance performance by avoiding unnecessary block editor assets
* New: Introducing Optimize Image Loading as a Beta feature - Apply default optimization strategies during content rendering for improved LCP
* Tweak: Allowed using TinyMCE Full Screen Toolbar with Editor Top Bar (#23463, #22873)
* Tweak: Make the Container element extendable (props @HadyShaltout)
* Tweak: Added a FAQ Schema support in Accordion widget
* Tweak: Added VideoPress integration for self-hosted videos in Video widget
* Tweak: Implemented accessibility improvements in Accordion widget
* Tweak: Upgraded minimum required PHP version to 7.3
* Fix: Responsive settings for Templates wouldn't save completely when Additional Custom Breakpoints feature is activated (#19394, #22829)
* Fix: Error messages appear on Theme Builder when using PHP 8.X (#22991)
* Fix: Disabling options in URL Control does not hide the options icon in External URL field (#11214)
* Fix: UI Glitch in widget Style and Advanced tabs (#23402)
* Fix: PHP Memory Limit displayed in System Info wasn't accurate in various scenarios

#### Pro version

* Tweak: Added AJAX pagination option or seamless page navigation between content in Loop Grid widget (#1284)
* Tweak: Introduced Individual Pagination option when multiple Post widgets are on the same page, providing enhanced control over pagination
* Tweak: Introduced Individual Pagination option when multiple Loop Grid widgets are on the same page, providing enhanced control over pagination
* Tweak: Added a None option to the breakpoint options in Menu widget
* Tweak: Added a horizontal scrolling option in Menu widget
* Tweak: Upgraded minimum required PHP version to 7.3
* Tweak: Improved accessibility when minimize button is disabled in Table of Content widget
* Fix: Table of Content widget without icons displays `undefined` error (#17523, #17768, #18463)
* Fix: Links inside Loop Carousel are not working on initial load in Menu widget (#23705)
* Fix: Popups accessible navigation adds outlines to the wrong first focusable element
* Fix: Dropdown area aligned to the left side of the screen if contains Tabs widget in Menu widget
* Fix: Content horizontal position not aligning correctly when used with carousel widgets in Menu widget
* Fix: Accessibility errors in PageSpeed Insights in Table of Contents widget

-----

## Make An Impact: Share Feedback Before Features Are Released

We’re always looking to get to know Elementor customers on a deeper level. If you’re interested in meeting with our team and sharing your insights, feedback, and feature requests, you’re welcome to sign up for some quality time with us [here](https://elementor.com/pages/joinresearch/).

### How to Install the Beta Version
If you still haven't enabled beta testing in Elementor, please follow these steps:

- Go to Elementor > Tools > Versions
- Enable the beta tester feature
- Go to Dashboard > Updates and update to the latest beta version.
- This will set you up to test the new Elementor Free and Pro 3.17 Beta.

----

## Notes

As we make changes to our GitHub in an effort to keep it organized, improve the way we deliver information to you, and our ability to review your feedback, please follow the instructions below when reporting bugs, or making feature requests:

- To report a bug related to beta versions - open a new [Bug report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml)
- For feature requests - open a new [Feature request discussion](https://github.com/orgs/elementor/discussions/new?category=feature-request)
- For other comments specifically about this beta version - comment here

Designated time for release: `October 24, 2023 (Estimated)`

Many thanks for your support and help!

Cheers,
_Elementor Team_

**Comment by 79ho3ein** [comment]
Nice update. But it would be much better if you introduced pagination as a new widget.

**Reply by nicholaszein** [reply]
Hey @79ho3ein!

I'm glat to see you here again! 💪

Thank you for taking the time to share your feedback!

After much deliberation we decided to go against adding a separate widget for pagination. 

Although it sounds a good idea, it also adds extra complexibility both in terms of UI & UX for web creators and visitors, but also in terms of code logic, and opens up multiple points of failure that would require extensive implementation, and maintenance, and could potentially cause issues in the long run, particularly in terms of performance of the Editor and in the front-end. 

We aim to simplify the building, and viewing experience of both Web Creators, and visitors alike without losing functionality (powerful, but easy to use, and intuitive). I think the way it is right now brings a good balance to this equation.

We will of course monitor user feedback and if we see a wider necessity, we will definitely rethink this strategy.

🙂

**Comment by DAnn2012** [comment]
Hi @nicholaszein

I kindly ask you if you think it is appropriate to include this modification of mine https://github.com/elementor/elementor/pull/23236, which is already approved by a reviewer, in the next version 3.17, so you don't have to wait any longer to see it included in version 3.18.

Thank you.

**Reply by nicholaszein** [reply]
Hello @DAnn2012!

Thank you for reaching out! 🙏

Please understand that when betas are released, they are already closed. The focus is to fix issues with the current beta, not to add more features or include other fixes, unless decided otherwise.

We really appreciate your contributions, and we've been adding them as our internal testing and QA is is finished. We can't rush these things though. We understand it might be frustrating to wait, but we want to ensure everything works to avoid breaking changes (which is something that even with all the testing in the world can still happen). 

I personally think your contributions are awesome! Your passion and willingness to help make and shape the future of Elementor are really admirable. Thank you for contributing! 🙏 

I'll check with the team to see what is the status on this PR, and ask them to expedite if possible. But please don't bet on it being merged to this version.

We ask for your patience as we decide on when and what to add to future versions.

**Comment by jlemonz** [comment]
Thanks for the update. Can a title text be added to the new rating widget? Just like the old one. Without that it is hard to add text infront of the rating.

**Reply by nicholaszein** [reply]
Hi @jlemonz,

Thanks for reaching out. You can use the Heading Title widget instead.

**Reply by jlemonz** [reply]
This is not a good solution. Now I need to add another container and an header widget and  try to place it nicely before the rating widget. Why just add the title field like the old version. Lot less work and tuning.

**Reply by nicholaszein** [reply]
Hey @jlemonz,

you don't really need to add another container. You can place the Heading Title widget inside the same container as the Rating widget, and then set your directions.

**Reply by jlemonz** [reply]
Yes if you want a single line on your page with a title and rating yes. But most of the times a star widget is used with other widgets in a container. SO to have the title next to it i have to add another container in a container.

**Reply by doronwolf** [reply]
If you look into the code of the old widget you will see this ability (to have title and rating placed in the same 'box') was achieved in a very similar way (using an extra div) so you are not affecting any performance by using a container to keep a title next to the rating and actually making it simpler and better for those who don't need it (since many times the rating widget is inside another container anyway)

**Comment by 79ho3ein** [comment]
I click on the container widget, but it is added as a grid. Even if the grid is disabled, this problem still occurs.

**Reply by nicholaszein** [reply]
Hey @79ho3ein can you show us an example in a video? I'm definitely not experiencing anything remotely close to what you are reporting. This got me really curious.

**Reply by 79ho3ein** [reply]
Hi @nicholaszein,

If this bug doesn't occur to you, I guess I accidentally converted the container to a grid earlier and then saved it as default. However, this can be problematic. I think, it is better for container and grid widgets, the option "Container Layout" does not follow the ability to save as default. Because it may cause problems.

https://github.com/elementor/elementor/assets/77163248/b1735324-8693-4247-9173-c463b57abd1f

**Reply by nicholaszein** [reply]
Hey @79ho3ein!

Thanks for the detailed video.

We are aware of this issue and we are working on a fix to be released in a future version! 💪

**Comment by shai707** [comment]
Great update.
Thanks Elementor team!⚡️⚡️

**Comment by fabigrafo** [comment]
Okay, so it's a relatively good, albeit small, update. Nothing was said about the visual changes in the editor top bar. What are the reasons? Unfortunately now everything seems to be a bit out of proportion. However, I don't think the larger elements are necessarily bad. But sometimes it's too big.

Maybe you should move the editor to the 2nd phase? When will the new panels come that will be removed from the left side?

It's been in the transition phase for 3 or 4 main versions and they always say that the eye eats with it. Currently the editor is very inconsistent. New things and lots of old things combined across multiple versions unfortunately seems amateurish.

Also, the dropdown next to “Publish” is ALWAYS pink. Also since a few versions. How long does this condition last approximately? Until then, unfortunately, I will only use the new things on my test environment. Until it is mature and makes sense.

However, some things from this update are fine so far. I'm not really interested in AI in a website editor, but the new "Star Rating" widget works perfectly. But I have a question about AJAX loading. Is this also set this way for the taxonomy filter? Because I have the feeling that it loads much faster and more beautifully. BUT, is there a way to customize the transition?

Thanks in advance!

**Reply by fabigrafo** [reply]
Even if the direction is right, everything moves very slowly. Unfortunately.

**Reply by nicholaszein** [reply]
Hey @fabigrafo!

Nice to see you here! 🙂 

> _**Nothing was said about the visual changes in the editor top bar. What are the reasons? Unfortunately now everything seems to be a bit out of proportion. However, I don't think the larger elements are necessarily bad. But sometimes it's too big.**_

There was no change in the Top Bar. It still remains the same as the previous versions. Are you sure you don't have a zoom or something set differently on your side?

> _**Maybe you should move the editor to the 2nd phase? When will the new panels come that will be removed from the left side?**_
> 
> _**It's been in the transition phase for 3 or 4 main versions and they always say that the eye eats with it. Currently the editor is very inconsistent. New things and lots of old things combined across multiple versions unfortunately seems amateurish**_

Unfortunately, we can't comment about our plans for the editor for the time being.

> _**Also, the dropdown next to “Publish” is ALWAYS pink. Also since a few versions. How long does this condition last approximately? Until then, unfortunately, I will only use the new things on my test environment. Until it is mature and makes sense.**_

This is the expected behavior. It was switched to be always pink on purpose for a better user experience. Users were having trouble realizing the dropdown was there.

> **_But I have a question about AJAX loading. Is this also set this way for the taxonomy filter? Because I have the feeling that it loads much faster and more beautifully._**

It's supposed to work with and without the ajax loading, with individual widgets, or with a single one. But we would love to know your feedback about using them. Please, try it out and follow these two instructions and let us know:
- https://github.com/orgs/elementor/discussions/23925#:~:text=an%20offset%20value.-,How%20to%20Test%20It%3A,-Ensure%20your%20website
- https://github.com/orgs/elementor/discussions/23925#:~:text=on%20the%20page.-,How%20to%20Test%20It%3A,-Ensure%20your%20website

> _**BUT, is there a way to customize the transition?**_

Currently that's not possible. However, if you think this should be an Elementor feature, please submit a new [🚀 Feature Request](https://github.com/orgs/elementor/discussions/new?category=feature-request), and we will consider it in the future if there is enough demand (votes) from other users.

Thanks for the feedback! 🙏

**Comment by contrastxxx** [comment]
Hello, thank you for another update, I like elementor very much and I'm still looking forward to the next news, but I've already written it several times and I still haven't received an answer, in the new TOP BAR disappeared the possibility of testing responsiveness in Desktop resolution, without it I can't use the new TOP BAR, thank you.

![image](https://github.com/elementor/elementor/assets/68115204/fe2bd8bb-7c9f-48bb-9447-d28a25bacdc2)

**Reply by nicholaszein** [reply]
Hello @contrastxxx!

Thank you for showing interest in our new version!

Please, remember to keep the conversation about this beta. Your feedback about the features being discussed here is crucial for releasing a stable and reliable plugin. We are eager to know about your experience with these new features. Thanks 🙏

If you wish to join the conversation, there is already a similar request submitted by someone here #22432

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

Kind regards

**Comment by portrayaloflife** [comment]
These look like some awesome new features. Will the role manager functionality be restored in this release?

**Reply by nicholaszein** [reply]
Hello @portrayaloflife,

The report on [this issue](https://github.com/elementor/elementor/issues/23696) is still being investigated. We will update the issue when whe have a solution for it.

Meanwhile, try to refresh the editor. It seems to be a workaround.

Please, remember to keep the conversation about this beta. Your feedback about the features being discussed here is crucial for releasing a stable and reliable plugin. We are eager to know about your experience with these new features. Thanks 🙏

**Comment by donwattz1459** [comment]
The mega menu seems to be broken in beta 1. It is expanded, CSS doesn't seem to be working either. 

https://dev.icisf.org/

**Reply by nicholaszein** [reply]
Hi @donwattz1459.

Please clear all levels of cache, especially server-side (especially static and object cache). Also, make sure the **`Improved CSS Loading`** experimental feature is set to **`Inactive`** in your **`Dashboard`** > **`Elementor`** > **`Settings`** > **`Features`** page. Then **`Regenerate CSS & Data`** in your  **`Dashboard`** > **`Elementor`** > **`Tools`** > **`General`** page.

**Reply by donwattz1459** [reply]
The improve css loading was the culpret. Thanks!

**Comment by sinanisler** [comment]
accordion nesting and other nesting features still freezing like hell in editor.

when too many containers and nested containers made in same page it is becoming impossible to edit. it feels like there is infinite loop in js that cant render cant solve it...

please make more tests

**Reply by nicholaszein** [reply]
@sinanisler,

please if you are experiencing performance issues or bugs with the nested Accordion widget, we would love to know more about it.

Submit a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.

Thanks

**Reply by sinanisler** [reply]
I already reported this nesting problems   🙏🙏

https://github.com/elementor/elementor/issues/23704#issuecomment-1722526028

**nested carousels, nested accordion, nested loops ALL EFFECTED.** editor performance/rendering/lag slow...

**Comment by pauserratgutierrez** [comment]
Nice update! Can we use the new Star Rating with dynamic data?

**Reply by fabigrafo** [reply]
Yes!

**Reply by SolunaCG** [reply]
But it´s not possible to grab reviews from Google with it, is it? That´s actually what I was hoping for when I read "a dynamic rating widget...". 
In our company our clients always want us to implement their clients ratings from Google and we always have to use a 3rd party widget for that which is freemium unfortunately.

**Reply by nicholaszein** [reply]
Hi @SolunaCG.

Google Reviews requires embedding and has a very strict rule on customization. The alternative is to implement API integration, which would also require dedicated code and customization to make everything work. This is a huge endeavor, that can't be added without thorough consideration and planning.

This new widget was created with the simple task of replacing the old one with better markup, compatibility with Google Rich Snippets, compatibility with the new builder functionalities, and more customization that has been asked by many users. It's not meant to be a full-fledged review system with deep dynamic integration and extra functionality.

If you think this should be an Elementor feature, please join the conversation in #14651 and we will consider it in the future if there is enough demand (votes) from other users.

Kind regards

**Reply by SolunaCG** [reply]
Hey @nicholaszein ,

thank you so much for your clarification! I wasn´t really sure what this update was aiming for which is why I was a bit confused about the new rating widget. 

I just joined the conversation you mentioned for a "Google Review Widget". Maybe we´ll have this in the future 😊 . 

Kind regards!

**Comment by Joliamendi** [comment]
Elementor team, is there still no fix for [this bug](https://github.com/elementor/elementor/issues/22431)? Megamanu is still unusable. At least for me, who use two megamenu in the header.

**Reply by nicholaszein** [reply]
Hey @Joliamendi!

This fix for this issue is in progress, and will be released in a future patch release.

Please keep the conversation about this beta. We would love to know your feedback about the new new tweaks we introduced! 🙏

**Reply by tobias992** [reply]
@nicholaszein 
There is a very easy solution - don't release buggy alpha widgets! But this is elementor mentality - "before we have nothing new to release, we bring buggy bull****". Why there is no place for FA6 in this release? It's a joke...

And yes... feel free to remove my comment. This is how elementor handles criticism.

**Reply by nicholaszein** [reply]
@tobias992 there is a difference between constructive criticism and disrespect.

We appreciate your feedback but there is no need to curse and swear.

Using these words won't help with your argument, they will actually have the opposite effect.

Please edit your comment.

Thank you

**Reply by tobias992** [reply]
@nicholaszein 
No. Remove it. Me and also a lot of other people are done with your customer mentality. Take care of your customers and also listen to them. Because WE pay your bills. The best example is FA6 support. What are you doing all the days, just thinking about new buggy alpha widgets? These days i created a support ticket in customer area because of a big bug with elementor product page template builder with official woo subscription plugin. I send screenshots with elementor problem code. Just 5 seconds work for your dev team to fix the problem. Typical elementor answer "Thank you, we are working on it". This bug has been there since month! And for this quality work you want to have our money. Sorry, but i am done with your company. Bye!

**Reply by donwattz1459** [reply]
Couldn’t you just use two containers and a mega menu in each? It’s what I’m doing.

**Reply by nicholaszein** [reply]
@tobias992,

I think you might be mistaken. All we've done thus far is listen to user feedback and add the [features that were most requested](https://github.com/elementor/elementor/issues?q=is%3Aissue+is%3Aclosed+label%3A%22%E2%9C%85+solved%22%2Cstatus%2Fmerged+label%3Arequest%2C%22request%2Fwidget+%2A%22%2Crequest%2Ffeature%2Crequest%2Fenhancement%2Crequest%2Fworkflow%2Crequest%2Fintegration+sort%3Acomments-desc) by our users. We also have actively solved [multiple bugs reported by lots of users](https://github.com/elementor/elementor/issues?q=is%3Aissue+is%3Aclosed+label%3A%22%E2%9C%85+solved%22%2Cstatus%2Fmerged+label%3Abug+sort%3Acomments-desc) consistently.

We do care about our users very much, and we follow all bug reports and requests closely. Monthly there are around [100 new issues](https://github.com/elementor/elementor/issues?q=is%3Aissue+created%3A2023-08-01..2023-08-31) added to this repository, and [around 70 new feature requests](https://github.com/orgs/elementor/discussions?discussions_q=is%3Aopen+created%3A2023-08-01..2023-08-31). Our team works tirelessly to solve the most pressing matters, and balance between adding the most requests features. 

And these are just the stats here on GitHub. We also follow closely all that is reported through our other channels as well, which if you haven't guessed already is a bigger number. So far there have been no features that have been added without user request. We have a dedicated team that works day and night to fix bugs and deliver all the changes we need to deliver on time. We have a very tight development cycle and we release bug fixes and patches almost every week.

As you can see, the points you've made do not reflect reality.

But I understand your frustration. You want the things you want, and you think they should be prioritized. That's completely understandable, and we do take feedback like yours as an indicator of how we should prioritize our tasks. However, swearing, cursing, and making accusations are not taken into account as a signal to prioritize tasks. And we do take disrespect very seriously here. If you have complaints to make, please reach out to Elementor Support. This is not the place to complain about the Elementor company. We are here to discuss this Beta version, and answering your rants takes time from working on the very issues you are complaining about. 

Let's not forget that Elementor is a Free open-source project, used on millions of websites, only a small fraction of these use the paid Pro version. We are also humans who need to eat, sleep, and pay the bills. Please don't assume we don't care because this is a false accusation.

Take care

**Comment by avivu** [comment]
Hi all,

We just released Elementor v3.17 Beta 2 for the Free version.


The version include the following:


#### Free version

* Fix: Widget panel appears empty for Editor role with "Access to edit content only" permission (#23696)
* Fix: Improved code security enforcement in SVG Icon control


–

Thank you!

**Reply by fabigrafo** [reply]
This are the same changelog like for the 3.16.5, correct?

**Comment by portrayaloflife** [comment]
Yay!!

**Comment by JoshPNG** [comment]
Will there be a fix for the issue here: https://github.com/elementor/elementor/issues/22368

Also are you aware of the loop carousel causing a longer than normal LCP when using autoplay? Page speed metrics will count the LCP until the 2nd slide of the carousel.

**Comment by aitorburgui** [comment]
When can the shopping cart icon be changed to a personalized one, as has happened with the menu icon?

**Comment by sekhsoyebali** [comment]
Please fix this Bug in next update - https://github.com/elementor/elementor/issues/22991

**Comment by avivu** [comment]
Hi all,

We just released Elementor v3.17 Beta 3 and Elementor Pro Beta 2.


The versions include the following:


#### Free version


* Tweak: Changed Gutenberg Asset Loading experiment naming to Optimized Gutenberg Loading
* Fix: Icons are not breaking into lines when there is not enough space in the container
* Fix: General UI tweaks and fixes in AI History feature



#### Pro version

* Fix: Hover behavior issues on menu items in Menu widget (#23430, #22431)
* Fix: AJAX Pagination with Previous and Next options doesn't work on archive and single templates in Loop Grid widget

–

Thank you!

**Comment by avivu** [comment]
Hi all,

We just released Elementor v3.17 Beta 4 and Elementor Pro Beta 3.


The versions include the following:


#### Free version


* Fix: Tab title alignment issues with Horizontal Scroll for logged-out users in the Tabs widget
* Fix: Improved code security enforcement in Dynamic Tag Fallback control



#### Pro version

* Fix: AJAX Pagination with Previous and Next options doesn't work on archive and current query in Loop Grid widget


–

Thank you!

**Comment by fabigrafo** [comment]
Will the release of this Version be on schedule?

**Comment by avivu** [comment]
Hi All,

### General update:

The release date has been rescheduled for tomorrow, estimated for October 24, 2023, as we are in the final stages of preparing the version's release.

--

Thank you!

**Comment by SolunaCG** [comment]
Will #23870 be fixed in this release?

**Reply by fabigrafo** [reply]
On last day, if this is not listed in this thread, then not.

**Comment by lensbreak** [comment]
Widget for ACF Repeater Fields, PLEASEE !!

**Reply by sinanisler** [reply]
stay away from the acf repeaters as much as you can only use it if you have to.
it kills the performance more you add more slow it makes your db calls.

I am not even talking about backend editing thats another story... ACF team needs to rewrite this or something.

**Comment by nmbrthirteen** [comment]
there's no ajax pagination in my loop grid. any idea how I can fix that?

![image](https://github.com/elementor/elementor/assets/61732491/4c68fbe6-6666-4921-a876-ab2ae17db69a)

**Reply by nicholaszein** [reply]
Hello @nmbrthirteen,

please make sure you have both Elementor v3.17.1 and Elementor Pro 3.17.0 (as of this comment) installed. It is likely you have an older version of either Elementor Core or Pro installed.

**Comment by alican532** [comment]
When I click on theme builder, I get the following error: Error: Action data is missing or empty. Action: app_site_editor_template_types

**Reply by doronwolf** [reply]
this is a known issue and the fix is already out - please update to 3.17.1 🙂

**Reply by alican532** [reply]
Thank you

**Reply by nicholaszein** [reply]
This issue was solved with the release of Elementor Core version 3.17.1.

**Comment by nicholaszein** [comment]
Hey, beta testers! 👋 

### 📢 Elementor Core and Elementor Pro **v3.17** have been released! 🥳 

Thank you for contributing to this beta, and for sharing your feedback.

Special thanks go to our contributors who helped make this release possible:

- @79ho3ein
- @DAnn2012
- @jlemonz
- @shai707
- @fabigrafo
- @contrastxxx
- @portrayoflife
- @donwattz1459
- @sinanisler
- @pauserratgutierrez
- @SolunaCG
- @alican532

Thank you! See you on the next one! 💪 🚀 

✅ Feel free to check it out and update your plugins to the new version!

Cheers 🥂

> Changelogs of Elementor Core and Pro
> [Elementor Core](https://wordpress.org/plugins/elementor/#developers)
> [Elementor Pro](https://elementor.com/pro/changelog/)
